packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -126,9 +127,25 @@ async function handler(req: CustomRequest) {
     throw new HttpError({ statusCode: 400, message: "User not found" });
   }
 
-  // If it's just an attendee of a booking then just remove them from that booking
-  const result = await handleSeatedEventCancellation(req);
-  if (result) return { success: true };
+  // get webhooks
+  const eventTrigger: WebhookTriggerEvents = "BOOKING_CANCELLED";
+
+  const subscriberOptions = {
+    userId: bookingToDelete.userId,
+    eventTypeId: bookingToDelete.eventTypeId as number,
+    triggerEvent: eventTrigger,
+    teamId: bookingToDelete.eventType?.teamId,
+  };
+  const eventTypeInfo: EventTypeInfo = {
+    eventTitle: bookingToDelete?.eventType?.title || null,
+    eventDescription: bookingToDelete?.eventType?.description || null,
+    requiresConfirmation: bookingToDelete?.eventType?.requiresConfirmation || null,
+    price: bookingToDelete?.eventType?.price || null,
+    currency: bookingToDelete?.eventType?.currency || null,
+    length: bookingToDelete?.eventType?.length || null,
+  };
+
+  const webhooks = await getWebhooks(subscriberOptions);
```

Comment:
That's the code that got removed below. Brought it up here so I can reuse it for seated events